catalogs/catalog-jdbc-mysql/build.gradle.kts

```diff
@@ -25,30 +25,36 @@ plugins {
 }
 
 dependencies {
-  implementation(project(":api"))
-  implementation(project(":catalogs:catalog-common"))
-  implementation(project(":catalogs:catalog-jdbc-common"))
-  implementation(project(":common"))
-  implementation(project(":core"))
+  implementation(project(":api")) {
+    exclude(group = "*")
+  }
+  implementation(project(":catalogs:catalog-common")) {
+    exclude(group = "*")
+  }
```

Comment:
Why only mysql relies on "catalog-common"?

Reply:
Only MySQL use `MysqlConstants.GRAVITINO_AUTO_INCREMENT_OFFSET_KEY` and support the feature `auto_increment` now.